eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -146,6 +150,13 @@ jobs:
     - name: sourcesPath
       value: $(vmrPath)
 
+  - ${{ if ne(parameters.buildPass, '') }}:
+    - name: _artifactsSuffix
```

Comment:
FWIW, this naming convention is different than what is currently being used.  IMO it should be consistent.  Either use the current convention or convert everything (preferably in a separate PR).